packages/client/hmi-client/src/components/pyciemss/tera-pyciemss-cancel-button.vue

```diff
@@ -12,10 +12,14 @@ const props = defineProps<{
 	simulationRunId?: string | string[];
 }>();
 
-const disabled = computed(() => props.simulationRunId === '');
+const isCancelling = ref(false);
+const disabled = computed(
+	() => props.simulationRunId === '' || props.simulationRunId?.length === 0 || isCancelling.value
+);
 
 const cancelSimulation = async () => {
 	if (!props.simulationRunId) return;
+	isCancelling.value = true;
```

Comment:
When are you setting it back to `false`?

Reply:
Nice catch. I feel like it should be at the start of `pollAction`?

Reply:
Thank you totally missed this